apps/cli/cli/link.md

```
@@ -54,14 +54,21 @@ Clear feedback:
+Error: Path not found: missing.md
+
+$ mkdir dir && git-mind link README.md dir
+Error: Not a regular file: dir
 
 $ git-mind link 
 Usage: git-mind link <source> <target> [--type <type>]
 Types: implements, references, depends_on, augments
 ```
 
 Helpful errors:
+- Specific path names in messages
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)